generator/import_tracker.go

```diff
@@ -58,7 +58,8 @@ func golangTrackerLocalName(tracker namer.ImportTracker, t types.Name) string {
 			// This name collides with some other package
 			continue
 		}
-		return name
+		// This name can be a reserved identifier or a keyword
+		return "_" + name
```

Comment:
does this prefix every identifier? Can we have a blacklist instead?

Reply:
Yes, this prefixes every package name in an import declaration. We can certainly prefix only reserved identifiers although I don't see the value in doing that over the simpler rule of prefixing every identifier.